var/spack/repos/builtin/packages/py-colorlog/package.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyColorlog(PythonPackage):
+    """A colored formatter for the python logging module"""
+
+    homepage = "https://github.com/borntyping/python-colorlog"
+    url      = "https://pypi.org/packages/source/c/colorlog/colorlog-4.0.2.tar.gz"
+
+    depends_on('py-setuptools', type='build')
```

Comment:
I like to put the versions above the dependencies since the versions change more frequently than the dependencies.